config/jobs/kubernetes-sigs/structured-merge-diff/structured-merge-diff-ci.yaml

```diff
@@ -41,3 +41,22 @@ periodics:
   annotations:
     testgrid-dashboards: sig-api-machinery-structured-merge-diff
     testgrid-tab-name: ci-gofmt
+- name: ci-structured-merge-diff-benchmark
+  interval: 12h
+  labels:
+    preset-service-account: "true"
+  decorate: true
+  spec:
+    containers:
+    - image: golang:latest
+      command:
+      - go
+      args:
+      - run
+      - k8s.io/test-infra/pkg/benchmarkjunit
+      - --log-file=$(ARTIFACTS)/benchmark-log.txt
+      - --output=$(ARTIFACTS)/junit_benchmarks.xml
+      - ./...
```

Comment:
This won't work, the job isn't configured to check out any code and it doesn't implicitly check out anything because it is periodically triggered (not implicitly associated with a git ref). See how `extra_refs` is used above.

Reply:
Oops, done: added back `extra_refs`.